File: tfr1/heimadæmi/hd9/d1/Fibonacci.java
public class Fibonacci {
    public static void Fib(int lo, int hi) {
        int a = 0; int b = 1;
        for(int i = 0; i <= hi; i++) {
            if (a > lo && a < hi)
                System.out.print(a + " ");
            int sum = a + b;
            a = b;
            b = sum;
        } System.out.println();
    }
    public static void main(String[] args) {
        int N = Integer.parseInt(args[0]);
        int M = Integer.parseInt(args[1]);
        Fib(N,M);
    }
}

File: tfr2/hd/h1/d1/Fibonacci.c
#include <stdio.h>
int fib(int x, int y) {
    int a = 0; int b = 1;
    for(int i = 0; i <= y; i++) {
        if (a > x && a < y) 
                printf("%d\n", a);
        int sum = a + b;
        a = b;
        b = sum;
    }
}

int main() {
    fib(5, 500);
}
